dist/index.html

```diff
@@ -1,15 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <title>Backbone Baseline</title>
-  </head>
-  <body>
-    <main id="main-content">
-      
+<head>
+  <meta charset="utf-8">
+  <title>KT MOVIE STORE</title>
+</head>
+<body>
+
+  <header>
+    <a><h1 id="logo" class="logo large-3 medium-3 columns"> Kay Kay and Tay Tay Movie Rentals </h1></a>
+    <ul class="menu large-4 medium-4 columns">
+      <li class="pageDescription">Use this dashboard to browse and add movies. That's it.</li>
+    </ul>
+
+    <ul id="search" class="menu align-right large-5 medium-5 columns">
+      <li><input type="text" name="queryTerm" placeholder="Movie Title" id="search-box" /></li>
+      <li><a id="search-button" type="button" value="Search">Search</a></li>
+    </ul>
+  </header>
+
+  <main id="main-content">
+
+      <section id='movie'></section>
+      <section id='movie-list'></section>
+      <!-- Templates -->
+
+      <script id="movie-template" type="text/template">
+        <h2><%- title %></h2>
+      </script>
+
+      <script id="movie-info-template" type="text/template">
+        <h3> <%- title %></h3>
+        <img src="https://image.tmdb.org/t/p/w185/<%- image_url %>" />
```

Comment:
You probably need an `if` statement here to get the images to show for IMDB movies, or you could work on the API end.